makefiles/info-global.inc.mk

```diff
@@ -27,6 +29,7 @@ define board_missing_features
   include $(RIOTBASE)/Makefile.dep
 
   FEATURES_MISSING := $$(sort $$(filter-out $$(FEATURES_PROVIDED), $$(FEATURES_REQUIRED)))
+  FEATURES_MISSING += $$(addprefix !,$$(sort $$(filter $$(FEATURES_PROVIDED), $$(FEATURES_BLACKLIST))))
```

Comment:
Same here I would prefer adding another variable name and/or other if case for this check.
These sections are hard enough to read, so adding an explicit name simplify this.